flashinfer/cute_dsl/sparse/sm100_blk64/block_sparse_tile_scheduler.py

```
@@ -0,0 +1,269 @@
+        for obj in objs:
+            obj_values = cutlass.extract_mlir_values(obj)
+            values += obj_values
+            self._values_pos.append(len(obj_values))
+        return values
+
+    def __new_from_mlir_values__(self, values):
+        obj_list = []
+        objs = [self.params, self._tile_idx]
+        if const_expr(self.params.scheduling_mode == SchedulingMode.CLC):
+            objs += [self._clc_scheduler, self._clc_pipeline, self._clc_consumer_state]
+        for obj, n_items in zip(objs, self._values_pos):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)